client/simple/vite.config.js

```diff
@@ -48,45 +48,20 @@ export default defineConfig({
   // - build.sourcemap
   // - esbuild.sourcemap
   // - css.preprocessorOptions.less.sourceMap
-
   css: {
-    devSourcemap: true,
-    preprocessorOptions: {
-      less: {
-        // FIXME: missing CCS sourcemaps!!
-        sourceMap: {
-          outputSourceFiles: true,
-          sourceMapURL: (name) => {
-            const s = name.split("/");
-            return `${s[s.length - 1]}.map`;
-          }
-        }
-        // env: 'development',
-        // relativeUrls: true,
-        // javascriptEnabled: true,
-      }
-    }
+    devSourcemap: true
   }, // end: css
 
-  esbuild: {
-    // FIXME: missing CCS sourcemaps!!
-    sourcemap: true
-  },
-
   build: {
-    target: "modules",
+    target: ["chrome87", "edge88", "firefox78", "safari14"],
```

Comment:
@inetol the commit message says "[fix] theme/simple: api sync " .. what exactly was the issue that has been fixed by this commit? .. except the modified `build.target` option I can't see any more real changes .. e.g. `build.cssMinify` has by default the same as [build.minify](https://vite.dev/config/build-options.html#build-minify) value -> https://vite.dev/config/build-options.html#build-cssminify ..